overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -13436,7 +13514,7 @@
           "ismain:1": 1,
           "issilent:1": 0,
           "lockedprogress:1": 1,
-          "name:8": "Fly To The Moon",
+          "name:8": "We Choose To Go To The Moon",
```

Comment:
Title is too long and ambiguous. Original seems better, or maybe `Launch to the Moon`, or `Liftoff`.

Reply:
Changed to Liftoff.